spec/controllers/dashboard_controller_spec.rb

```diff
@@ -340,4 +341,8 @@ def expect_failed_login(flash = nil)
     expect(response.body).to match(/flash_msg_div/) if flash
     # expect(result.body.to match(/flash_msg_div.*replaceWith.*#{msg}/) if flash
   end
+
+  def browser_info(typ)
+    session.fetch_path(:browser, typ).to_s
```

Comment:
I believe there is code somewhere (perhaps in an initializer) to include MoreCoreExtensions::Shared::Nested into the session class.